azkaban-exec-server/src/main/java/azkaban/common/ServerUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+
+package azkaban.common;
+
+import azkaban.Constants;
+import azkaban.execapp.event.JobCallbackManager;
+import azkaban.utils.Props;
+import org.apache.log4j.Logger;
+
+/**
+ * Common utility methods for Azkaban Executor and Flow Container
+ */
+public class ServerUtils {
```

Comment:
Shall we keep this class in azkaban-common module?

Reply:
Since this class is intended for execution side server utils, I put it in common of exec-server. I suppose we can make it a common class for web,exec and FlowContainer.
I do not have any preference, if you think that it should be in azkaban-common, I can move it there.

Reply:
Sure. We can address in separate PR.
^^ @sshardool

Reply:
No need to do it in separate PR, I can do it in this PR only.